.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+on:
+  pull_request:
```

Comment:
Ensure we do not trigger two jobs on each, take a look at https://github.com/ansible/vscode-ansible/blob/main/.github/workflows/ci.yaml#L7-L11 which works correctly.

Reply:
I intentionally omitted filtering because it's useful to build pushes to branches that don't yet have a PR open. If we want to have PR-only builds, we first need to enforce branches from forks and forbid upstream branches.